torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -33,10 +33,12 @@ auto AccumulateGrad::acc_inplace(std::shared_ptr<Variable>& grad,
 
 auto AccumulateGrad::apply(const variable_list& grads) -> variable_list {
   // XXX: this method is not thread-safe!
-  check_input_variables("AccumulateGrad", grads, 1);
-  auto var = variable.lock();
+  check_input_variables("AccumulateGrad", grads, 1, 0);
   auto new_grad = grads[0];
 
+  if (!new_grad) return {};
```

Comment:
This can be null when a function has no gradient with respect to a Variable.
The thing is that the python wrapper transforms null gradients into a tensor full of 0s so this only happens for C functions hence this usecase not present before.